docs/csharp/language-reference/tokens/raw-string.md

```diff
@@ -0,0 +1,37 @@
+---
+description: "Raw string literals can contain any arbitrary text without the need for special escape sequences.You begin and end a raw string literal with a minimum of three double-quote characters."
+title: "Raw string literals = \"\"\""
+ms.date: 12/08/2022
+---
+# Raw string literal text - `"""` in string literals
+
+A raw string literal starts and ends with a minimum of three double quote (`"`) characters:
+
+:::code language="csharp" source="./snippets/raw-string-literal.cs" id="SingleLine":::
+
+Raw string literals can span multiple lines:
+
+:::code language="csharp" source="./snippets/raw-string-literal.cs" id="MultiLine":::
+
+The following rules govern the interpretation of a multi-line raw string literal:
+
+- Both opening and closing quote characters must be on their own line.
+- Any whitespace to the left of the closing quotes is removed from all lines of the raw string literal.
+- Whitespace following the opening quote on the same line is ignored.
+- Whitespace only lines following the opening quote are included in the string literal.
+
+You may need to create a raw string literal that has three or consecutive double-quote characters. Raw string literals can start and end with a sequence of at least three double-quote characters. When you string literal contains three consecutive double-quotes, you start and end the raw string literal with four double quote characters:
+
+:::code language="csharp" source="./snippets/raw-string-literal.cs" id="MoarQuotes":::
+
+If you need to start or end a raw string literal with quote characters, use the multi-line format:
+
+:::code language="csharp" source="./snippets/raw-string-literal.cs" id="InitialQuotes":::
+
+Raw strings can also be combined with [interpolated strings](./interpolated.md#special-characters) to embed ther `{` and `}` characters in the output string.
```

Comment:
It would be nice to mention the ability of adding multiple `$` sign so that a matching number of `{` and `}` is interpolation and smaller occurrences of `{` and `}` are interpreted as-is